sound/soc/sof/intel/apl.c

```diff
@@ -75,6 +75,7 @@ int sof_apl_ops_init(struct snd_sof_dev *sdev)
 
 		/* debug */
 		sof_apl_ops.ipc_dump	= hda_ipc4_dump;
+		sof_apl_ops.dbg_dump	= hda_ipc4_dsp_dump;
```

Comment:
is this works with the reference firmware? IPC4 SOF is not buildable for APL.

Reply:
It is only for zephyr build FW

Reply:
I will remove it for APL